run_test.py

```diff
@@ -70,6 +71,7 @@
             'cuda': 'cuda90',
             'cudnn': 'cudnn7-cuda9',
             'nccl': 'nccl2.0-cuda9',
+            'ideep': 'ideep4py==1.0.1',
```

Comment:
> Looking at Jenkins test logs, it seems pip install ideep4py has not been invoked. Am I doing something wrong?

If no custom Dockerfile modification is required, add iDeep to `requires` section.
`conf` is transformed into Dockerfile like this: https://github.com/chainer/chainer-test/blob/master/docker.py#L469

Reply:
Fixed.